README.md

```diff
@@ -76,10 +76,28 @@ config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true,  :mou
 
 After a 'vagrant reload' you will be prompted for your local machine password.
 
-#### Provisioning with user-data
+#### Provisioning with Ignition (VirtualBox Provider (default))
 
-The Vagrantfile will provision your CoreOS VM(s) with [coreos-cloudinit][coreos-cloudinit] if a `user-data` file is found in the project directory.
-coreos-cloudinit simplifies the provisioning process through the use of a script or cloud-config document.
+When using the VirtualBox provider for Vagrant (the default), Ignition is used to provision the machine. This uses a special plugin that is
+automatically installed when using the default Vagrantfile. The config options for the plugin are all prefixed with `config.ignition` and can
+be found in this Vagrantfile or in the README of the [plugin][https://github.com/coreos/vagrant-ignition].
```

Comment:
This link is invalid. It should be `[plugin](https://github.com/coreos/vagrant-ignition)`. FYI, I found this using [marker](https://crates.io/crates/marker).